src/soundio/soundmanagerutil.h

```diff
@@ -30,14 +30,21 @@
  *        Mixxx.
  */
```

Comment:
If you're touching that header and cpp file anyway, can you switch to `///` comments?

Reply:
Done. But this will add a lot of noise.